sections/utility.js

```diff
@@ -61,8 +61,8 @@ export default [
   {
     type: "method",
     method: "binToUuid",
-    example: "knex.fn.binToUuid(binaryUuid)",
-    description: "Convert a binary uuid (binary(16)) to a string uuid (char(36))",
+    example: "knex.fn.binToUuid(binaryUuid, ordered)",
+    description: "Convert a binary uuid (binary(16)) to a string uuid (char(36)).Ordered is boolean defaults to true.It should be the same in binToUuid and uuidToBin",
```

Comment:
Hi, thx for the doc.
Can add whitespace after "." ?
Also could be nice to describe the purpose of this boolean in the description, it's not so trivial why we add this.